File: android/src/main/kotlin/com/lahaus/iterable_flutter/LogUtils.kt
package com.lahaus.iterable_flutter

import android.util.Log
import io.flutter.BuildConfig

/**
 * Created by alex on 21/11/22.
 */
object LogUtils {
    var enabled: Boolean = true
    fun debug(message: String, tag: String = "IterableFlutterPlugin") {
        if (BuildConfig.DEBUG && enabled) {
            Log.d(tag, message);
        }
    }
}

File: ios/Classes/LogUtils.swift
//
//  LogUtils.swift
//  iterable_flutter
//
//  Created by Alex Queudot on 22/11/22.
//

import Foundation

struct LogUtils {
    static var enabled: Bool = true
    
    static func debug(message: String, tag: String = "SwiftIterableFlutterPlugin") {
        if enabled {
            print("\(tag): \(message)")
        }
    }
}
